nep-5.mediawiki

```diff
@@ -93,6 +93,12 @@ If the <code>to</code> address is a deployed contract, the function SHOULD check
 
 If the transfer is not processed, the function SHOULD return <code>false</code>.
 
+The function MUST call <code>onPayment</code> method AFTER fire the <code>Transfer</code> event. This call need the flag <code>IfExists</code> and the <code>amount</code> argument. If the receiver doesn't want to receive this transfer it SHOULD <code>throw</code> an exception.
```

Comment:
Now we have the ability to catch exceptions in the contract. So throwing an exception may not be able to cancel the transfer. ABORT must be called.